Use implicit types for printing (granting wish #10366). (#11261)
coq/coq: doc/sphinx/language/gallina-extensions.rst

  Adds blocks of implicit types with different specifications.

.. flag:: Printing Use Implicit Types

>   By default, the type of bound variables is skipped for printing when

Reviewer: ```suggestion
  By default, the type of bound variables is not printed when
```

>   the variable is associated to an implicit type which matches the

Reviewer: ```suggestion
  the variable name is associated to an implicit type which matches the
```
